### PR TITLE
Add concept mappings

### LINKED ISSUE #1: Concepticon mapping
The raw, unmapped concepts in this dataset are now available here https://github.com/lexibank/polyglottaafricana/blob/master/cldf/parameters.csv

Can any of the more experienced concepticon people help mapping it?

### PR DESCRIPTION
**Pull request by chrzyki**
A first step at concept mappings. Closes #1.

There are a number of specialised concepts and double definitions (then left unmapped). Happy to refine this if needed.

### CODE REVIEW

**Review by LinguList — COMMENTED** [review]
I guess this will be basically okay, however, what I'd suggest is to later merge this with the official list at concepticon, which we discussed. here, it turned out their glosses diverge a bit from the source at times, so we'd like to keep it, right? So the key will be to work from concepticon, while at the same time making sure to trace the parts here. This has in part already been done on the google doc.

File: etc/concepts.tsv [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+Five	Five	493	FIVE
+Tooth	Tooth	1380	TOOTH
+Tongue	Tongue	1205	TONGUE
+Throat	Throat	1346	THROAT
+Gullet	Gullet		
+Neck	Neck	1333	NECK
+Shoulder	Shoulder	1482	SHOULDER
+Arm	Arm	1673	ARM
+ArmBetweenShoulderandElbow	Arm Between Shoulder and Elbow		
+ArmBetweenElbowandWrist	Arm Between Elbow and Wrist
```
Comment (Kristina-Pianykh) [inline etc/concepts.tsv]: LOWER ARM?

File: etc/concepts.tsv [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+Mouth	Mouth	674	MOUTH
+Five	Five	493	FIVE
+Tooth	Tooth	1380	TOOTH
+Tongue	Tongue	1205	TONGUE
+Throat	Throat	1346	THROAT
+Gullet	Gullet		
+Neck	Neck	1333	NECK
+Shoulder	Shoulder	1482	SHOULDER
+Arm	Arm	1673	ARM
+ArmBetweenShoulderandElbow	Arm Between Shoulder and Elbow
```
Comment (Kristina-Pianykh) [inline etc/concepts.tsv]: UPPER ARM?

File: etc/concepts.tsv [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+Throat	Throat	1346	THROAT
+Gullet	Gullet		
+Neck	Neck	1333	NECK
+Shoulder	Shoulder	1482	SHOULDER
+Arm	Arm	1673	ARM
+ArmBetweenShoulderandElbow	Arm Between Shoulder and Elbow		
+ArmBetweenElbowandWrist	Arm Between Elbow and Wrist		
+Leg	Leg	1297	LEG
+OuterHandorHand	Outer Hand, or Hand		
+InnerHand	Inner Hand
```
Comment (Kristina-Pianykh) [inline etc/concepts.tsv]: Maybe PALM OF HAND?

File: etc/concepts.tsv [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+Drum	Drum	908	DRUM
+Firewood	Firewood	10	FIREWOOD
+Walkingstick	Walking-stick	1296	WALKING STICK
+Leaf	Leaf	628	LEAF
+Groundnut	Ground-nut	595	GROUNDNUTS
+Milk	Milk	635	MILK
+Cat	Cat	1208	CAT
+Pig	Pig	1337	PIG
+Bat	Bat	1793	BAT
+FowlHen	Fowl (Hen)
```
Comment (Kristina-Pianykh) [inline etc/concepts.tsv]: Maybe FOWL? There's a 'chicken / fowl' label in Carling-2017-112e-2 mapped to FOWL too.

File: etc/concepts.tsv [hunk 5]
```diff
@@ -0,0 +1,336 @@
...
+Quiver	Quiver	995	QUIVER
+Greegree	Greegree	2804	AMULET
+Sacrifice	Sacrifice	1103	SACRIFICE
+Hell	Hell	878	HELL
+Waterfresh	Water (fresh)		
+Gold	Gold	1369	GOLD
+Book	Book	963	BOOK
+NewMoon	New Moon	3691	NEW MOON
+Soap	Soap	788	SOAP
+Needle	Needle	1382	NEEDLE (FOR SEWING)
```
Comment (Kristina-Pianykh) [inline etc/concepts.tsv]: Not sure but I would leave it unmapped due to the lack of specificity. This is unlikely to be the case, but it still can refer to a needle of a pine tree or the like.

File: etc/concepts.tsv [hunk 6]
```diff
@@ -0,0 +1,336 @@
...
+Oldofpersons	Old (of persons)	2112	OLD (AGED)
+Oldofthings	Old (of things)	2113	OLD (USED)
+Ieatyam	I eat yam		
+witch	witch	824	WITCH
+silver	silver	759	SILVER
+Sheep	Sheep	1331	SHEEP
+myyoungerson	my younger son		
+Isew	I sew	1457	SEW
+Milkofcows	Milk (of cows)		
+Milkofwomen	Milk (of women)	2129	BREAST OR MILK
```
Comment (Kristina-Pianykh) [inline etc/concepts.tsv]: I would unmap it.

File: etc/concepts.tsv [hunk 7]
```diff
@@ -0,0 +1,336 @@
...
+Doorofwood	Door (of wood)		
+Milkfresh	Milk (fresh)		
+Milksour	Milk (sour)		
+Rootstump	Root (stump)	241	TREE STUMP
+Pigeontame	Pigeon (tame)		
+Lizardthecommononemale	Lizard (the common one) (male)		
+ThelargeredheadedLizardfemale	The large red-headed Lizard (female)		
+Leafgrass	Leaf (grass)		
+New	New	1231	NEW
+Oldpersons	Old (persons)	566	OLD (OF WOMAN)
```
Comment (Kristina-Pianykh) [inline etc/concepts.tsv]: I would also unmap this one. There is no indication that the concept refers exclusively to women.

File: etc/concepts.tsv [hunk 8]
```diff
@@ -0,0 +1,336 @@
...
+Seventeen	Seventeen	2453	SEVENTEEN
+Eighteen	Eighteen	1713	EIGHTEEN
+Nineteen	Nineteen	1714	NINETEEN
+Buttermelted	Butter (melted)		
+Camwood	Camwood		
+ChainFettersforneck	Chain (Fetters ?) (for neck)		
+Batlarge	Bat (large)		
+Pigeonwild	Pigeon (wild)		
+Yamwild	Yam (wild)		
+Rainwater	Rain (water)
```
Comment (Kristina-Pianykh) [inline etc/concepts.tsv:290]: Maybe map this one to RAIN (PRECIPITATION) as it clearly points to the substance while the one above map to RAIN OR RAINING?

**Review by Kristina-Pianykh — COMMENTED** [review]
I think I worked half-way through this concept list and I just checked it on the google doc. The structure (the order of the items) is different from the one in the book but I don't know which this precise list comes from.

**Comment by LinguList** [comment]
Do you have the PDF? I have the PDF, but I think @chrzykie may also have it?

**Comment by Kristina-Pianykh** [comment]
I have the scanned pages with the concept list. I can send it if you need.

File: etc/concepts.tsv (hunk 7 above)
Comment (chrzyki) [reply etc/concepts.tsv]: Was a mistake

File: etc/concepts.tsv (hunk 8 above)
Comment (chrzyki) [reply etc/concepts.tsv:290]: I think this is for food/consumption, e.g. a specific lexeme for water for food from rain.

**Comment by chrzyki** [comment]
Updated.

**Review by xrotwang — APPROVED** [review]
I think that's ok as a start - and refinement should then happen in a compatible way at concepticon.

**Pull request merged**